web_src/less/_repository.less

```diff
@@ -3011,6 +3018,11 @@ tbody.commit-list {
       pointer-events: none !important;
     }
   }
+  @media @mediaSm {
+    .svg {
+      display: none;
+    }
```

Comment:
The first question in my mind is that "is the `.svg` class too broad?`, it so general a name.

Reply:
Well, I'm not sure which other svg's are under the `.repo-buttons .ui.labeled.button` class? Sounds already pretty specific to me.

Reply:
OK, I see, another question is why only hide the `.svg`?